src/msvc_helper_main-win32.cc

```diff
@@ -113,12 +113,13 @@ int MSVCHelperMain(int argc, char** argv) {
     PushPathIntoEnvironment(env);
   }
 
-  char* command = GetCommandLineA();
-  command = strstr(command, " -- ");
-  if (!command) {
+  wchar_t* w_command = GetCommandLine();
```

Comment:
Can you instead call

```cpp
WideToUtf8(GetCommandLine());
```

 directly?

Reply:
Note that it's very important that you make sure `WideToUtf8` be modified to accept something *other* than std::wstring. Internally `WideToUtf8` only cares about the pointer and size. So make it accept a StringPiece instead. That way, you don't allocate a std::wstring unnecessarily.